state/initialize_test.go

```diff
@@ -123,6 +123,7 @@ func (s *InitializeSuite) TestInitialize(c *gc.C) {
 	controllerCfg := testing.FakeControllerConfig()
 
 	ctlr, err := state.Initialize(state.InitializeParams{
+		SSHServerHostKey: "dummy",
```

Comment:
Personal preference, I'd use `testing.SSHServerHostKey`

Reply:
Is it ok I leave this? It feels more in-line with what's already there